Source/SpaceParkTester2/UnitTest1.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SpacePark;
+
+namespace SpaceParkTester2
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
+        public void IsValidPerson_ValidInput_True()
+        {
+            var spacePark = new ParkingEngine();
+
+            var IsValid = spacePark.IsValidPerson("Luke Skywalker");
+
+            Assert.IsTrue(IsValid);
+
+        }
+
+        [TestMethod]
+        public void IsValidPerson_InvalidInput_False()
+        {
+            var spacePark = new ParkingEngine();
+
+            var IsValid = spacePark.IsValidPerson("Benjamin Ytterstr�m");
+
+            Assert.IsTrue(IsValid);
+        }
```

Comment:
Bra med Unit-test på metoder, fortsätt så med dem andra metoderna och de metoder som kommer läggas in.